api/api/log_api.go

```diff
@@ -42,38 +48,36 @@ func (l *LogController) ReadLog(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	res, err := l.LogService.ReadLog(&query)
-	if err != nil {
-		log.Errorf("Error while retrieving log %v", err)
-		InternalServerError(fmt.Sprintf("Error while retrieving log for container %s: %s", query.Name, err)).WriteTo(w)
-		return
-	}
+	logLineCh := make(chan string, 1000)
```

Comment:
Why `1000`?

Reply:
No longer using buffered channel here. Unbuffered channel is better suited for logging case so it doesn't need to wait until the limit of the buffer is reached to send the log.